client/main.lua

```diff
@@ -41,7 +41,8 @@ end
 local function getNearestVeh()
     local pos = GetEntityCoords(PlayerPedId())
     local entityWorld = GetOffsetFromEntityInWorldCoords(PlayerPedId(), 0.0, 20.0, 0.0)
-    local rayHandle = CastRayPointToPoint(pos.x, pos.y, pos.z, entityWorld.x, entityWorld.y, entityWorld.z, 10, PlayerPedId(), 0)
+    local rayHandle = CastRayPointToPoint(pos.x, pos.y, pos.z, entityWorld.x, entityWorld.y, entityWorld.z, 10,
```

Comment:
unnecessary change

Reply:
Thats pretty standard linting, done automatically upon me saving the file.
Not directly related to this export option, no. But we should be formatting our code correctly.